pyp5js/pyp5js.py

```diff
@@ -566,7 +572,10 @@ def loadPixels(*args):
     return _P5_INSTANCE.loadPixels(*args)
 
 def set(*args):
-    return _P5_INSTANCE.set(*args)
+    if len(args) == 1:
```

Comment:
Python's **set()** can be called w/ no args too! So it should be instead: `if len(args) <= 1:`

Reply:
Fixed!